web/src/popover_menus.ts

```diff
@@ -154,7 +176,7 @@ export const default_popover_props = {
                 phase: "beforeWrite",
                 requires: ["$$tippy"],
                 fn({state}) {
-                    const instance = state.elements.reference._tippy;
+                    const instance = $(state.elements.reference)._tippy;
                     const $popover = $(state.elements.popper);
```

Comment:
@Lalit3716 this won't work. We should be able to `_tippy` directly here since you already added it to HTMLElement.

Reply:
Would be better to do this check?
```js
if (!("_tippy" in state.elements.reference)) {
    return;
}
```